cmd/minikube/cmd/start.go

```diff
@@ -68,7 +68,7 @@ func runStart(cmd *cobra.Command, args []string) {
 		KubernetesVersion: kubernetesVersion,
 	}
 
-	fmt.Println(config.DiskSize)
+	fmt.Printf("minikubeVM Disk Size: %d", config.DiskSize)
```

Comment:
This should be removed - it was a bit of debug I left in by mistake & has been removed in recent commits.